systemtest/src/main/java/io/strimzi/systemtest/utils/VersionModificationData.java

```diff
@@ -0,0 +1,202 @@
+        this.urlFrom = urlFrom;
+    }
+
+    public void setOldestKafka(String oldestKafka) {
+        this.oldestKafka = oldestKafka;
+    }
+
+    public void setDefaultKafka(String defaultKafka) {
+        this.defaultKafka = defaultKafka;
+    }
+
+    public void setDeployKafkaVersion(String deployKafkaVersion) {
+        this.deployKafkaVersion = deployKafkaVersion;
+    }
+
+    public void setStartingKafkaVersion(String startingKafkaVersion) {
+        this.startingKafkaVersion = startingKafkaVersion;
+    }
+
+    public void setFeatureGatesBefore(String featureGatesBefore) {
+        this.featureGatesBefore = featureGatesBefore;
+    }
+
+    public void setFeatureGatesAfter(String featureGatesAfter) {
+        this.featureGatesAfter = featureGatesAfter;
+    }
+
+    public void setConversionTool(Map<String, String> conversionTool) {
+        this.conversionTool = conversionTool;
+    }
+
+    public void setImagesAfterKafkaUpgrade(Map<String, String> imagesAfterKafkaUpgrade) {
+        this.imagesAfterKafkaUpgrade = imagesAfterKafkaUpgrade;
+    }
+
+    public void setClient(Map<String, Object> client) {
+        this.client = client;
+    }
+
+    public void setEnvironmentInfo(Map<String, String> environmentInfo) {
+        this.environmentInfo = environmentInfo;
+    }
+
+    public void setToVersion(String toVersion) {
+        this.toVersion = toVersion;
+    }
+
+    public void setToExamples(String toExamples) {
+        this.toExamples = toExamples;
+    }
+
+    public void setUrlTo(String urlTo) {
+        this.urlTo = urlTo;
+    }
+
+    public void setImagesAfterOperatorDowngrade(Map<String, String> imagesAfterOperatorDowngrade) {
+        this.imagesAfterOperatorDowngrade = imagesAfterOperatorDowngrade;
+    }
+
+    @Override
+    public String toString() {
+        return "VersionModificationData{" +
+                "additionalTopics=" + additionalTopics +
+                ", fromVersion='" + fromVersion + '\'' +
+                ", fromExamples='" + fromExamples + '\'' +
+                ", urlFrom='" + urlFrom + '\'' +
+                ", oldestKafka='" + oldestKafka + '\'' +
+                ", defaultKafka='" + defaultKafka + '\'' +
+                ", deployKafkaVersion='" + deployKafkaVersion + '\'' +
+                ", startingKafkaVersion='" + startingKafkaVersion + '\'' +
+                ", featureGatesBefore='" + featureGatesBefore + '\'' +
+                ", featureGatesAfter='" + featureGatesAfter + '\'' +
+                ", conversionTool=" + conversionTool +
+                ", imagesAfterKafkaUpgrade=" + imagesAfterKafkaUpgrade +
+                ", client=" + client +
+                ", environmentInfo=" + environmentInfo +
+                ", toVersion='" + toVersion + '\'' +
+                ", toExamples='" + toExamples + '\'' +
+                ", urlTo='" + urlTo + '\'' +
+                ", imagesAfterOperatorDowngrade=" + imagesAfterOperatorDowngrade +
+                '}';
```

Comment:
This is fine, I would just add there new lines to make it more easier to read it.